docs/manual/install/index.rst

```diff
@@ -45,14 +45,14 @@ cairocffi
 ---------
 
 Qtile uses cairocffi_ with XCB support via xcffib. You'll need ``libcairo2``,
-the underlying library used by the binding.  You should be sure before you
-install cairocffi that xcffib has been installed, otherwise the needed
+the underlying library used by the binding.  You should **be sure before you
+install cairocffi that xcffib has been installed**, otherwise the needed
 cairo-xcb bindings will not be built.  Once you've got the dependencies
 installed, you can use the latest version on PyPI:
 
 .. code-block:: bash
 
-    pip install cairocffi
+    pip install --no-cache-dir cairocffi
```

Comment:
This is nice, we may want to add this to the error message that we print if cairocffi is not properly installed.

Reply:
Where would be the best place to print this error message?

Reply:
Ah, I was thinking in `setup.py`, but it looks like that error message already has this.